mapper/mapper.go

```diff
@@ -577,6 +577,12 @@ func CreateHierarchyPage(h hierarchyClient.Model, dst dataset.Model, f filter.Mo
 		title = h.Label
 	}
 
+	if p.Data.DimensionName == "Goods and Services" {
+		p.Type = "type"
+	} else {
+		p.Type = "area"
+	}
```

Comment:
This is fine, but would be a pain to extend - would be better off creating a map at the top (like dimensionTitleTranslator)

For example:

```go
var hierarchyBrowseLookup = map[string]string{
	"geography": "area",
}
```

and then have

```go
	var ok bool
	if p.Type, ok = hierarchyBrowseLookup[name]; !ok {
		p.Type = "type"
	}
```

which would default to "type" if the value does not exist in the map